website/thaliawebsite/management/commands/migratemedia.py

```diff
@@ -0,0 +1,45 @@
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+import logging
+import os
+import boto3
+from botocore.exceptions import ClientError
+
+
+class Command(BaseCommand):
```

Comment:
Maybe it is nicer to make this a pre or post migration signal?

Reply:
I am not entirely sure what you mean by such a signal. But technically this script is more a one time thing to transfer everything. Not even sure we need it, since the AWS CLI has the `sync` command that could do the same thing.